lib/Clam/SeaDsaHeapAbstraction.cc

```diff
@@ -433,28 +432,29 @@ void SeaDsaHeapAbstraction::initialize(const llvm::Module &M) {
       // Keep track of inconsistent memory regions (i.e., regions on
       // which caller and callee disagree)
       for (unsigned i = 0, e = readsC.size(); i < e; i++) {
-        readsB[i] = readsB[i] & readsC[i].second;
-	if (!readsC[i].second) {
-	  CLAM_WARNING("Caller and callee disagree on some read region type\n"
-		       << "Callsite=" << *CI << "\n"
-		       << "Caller=" << *CS.getCaller() << "\n");
-	}
+        readsB[i] = readsB[i] && readsC[i].second;
+        if (!readsC[i].second) {
+          CLAM_WARNING("Caller and callee disagree on some read region type\n"
+                       << "Callsite=" << *CI << "\n"
+                       << "Caller=" << *CS.getCaller() << "\n");
+        }
       }
       for (unsigned i = 0, e = modsC.size(); i < e; i++) {
-        modsB[i] = modsB[i] & modsC[i].second;
-	if (!modsC[i].second) {
-	  CLAM_WARNING("Caller and callee disagree on some modified region type\n"
-		       << "Callsite=" << *CI << "\n"
-		       << "Caller=" << *CS.getCaller() << "\n");
-	}	
+        modsB[i] = modsB[i] && modsC[i].second;
```

Comment:
this is an important change

Reply:
@caballa please double check. Crab overrides `&` so sometimes bit-operation is the right one, no matter what clang says

Reply:
Clang is right here